course/models.py

```diff
@@ -341,7 +338,7 @@ def get_visible(self, user=None):
         if not user or not user.is_authenticated:
             return self.filter(visible_to_students=True)
         if not user.is_superuser:
-            return self.filter(get_course_visibility_filter(user)).distinct()
+            return self.filter(get_course_visibility_filter(user) | Q(visible_to_students=True)).distinct()
```

Comment:
This lines might be a bit long. Could to add some newlines and try to keep line lengths within 100 or 80 characters. Operator would go in the start of line, thus

```python
return self.filter(
    get_course_visibility()
    | Q(....)
).distinct()
```

Not the perfect. You can also wrap everything in brackets and add new lines accordingly

Reply:
Sure, I wasn't sure how to indent it correctly so left it like that :)